src/html-conventions-lib/general-connectors-html-convention.xsl

```diff
@@ -501,7 +501,7 @@
         <xsl:variable name="targetName" select="$connector/target/role/@name"/>
         <xsl:sequence
             select="
-            if (($numberOfTags > 0) and not(boolean($targetName))) then
+                if (($numberOfTags > 0) and not(boolean($targetName))) then
```

Comment:
I don't think this logic is doing what it is supposed to do. 
Just get all tags and then for each tag check if it has a name.

Reply:
this rule is checking if the target role has tags but no name. This is not checking if a tag has a name. There is another checker for this